### ISSUE TITLE
how do I disable cache for React Native?

### ISSUE THREAD

**Issue by kdong007** [feats/babel-plugin-inline-import#15]
it keeps loading the older version of .graphql

**Comment by geyang** [comment]
exactly, I ran into exactly the same problem. This is a very serious issue.

In my case, the older version of markdown just doesn't get updated. We should fix this via a PR.

https://github.com/remarkjs/remark-frontmatter/issues/5
